include/deal.II/lac/arpack_solver.h

```diff
@@ -875,6 +969,23 @@ ArpackSolver::solve(const MatrixType1 & /*system_matrix*/,
 }
 
 
+
+template <typename VectorType, typename MatrixType1>
+inline void
+ArpackSolver::solve(const MatrixType1 &                A,
```

Comment:
wait a second, but why don't you provide the inverse operation for standard eigenvalue problem? Surely you can you shift-and-invert for `Ax=\lambda x`?

Reply:
I looked in the documentation and they only talk about shift-and-invert for the generalized problem

Reply:
that's strange, bu ok...